frontend/src/app/modules/email/email.service.ts

```diff
@@ -0,0 +1,108 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+import { environment } from 'src/environments/environment';
+import { SharedService } from '../shared/services/shared.service';
+import { CreateEntityModalData, CreateEntityPayload } from '../shared/components/entity-create-modal/entity-create.types';
+import { Entity } from '../shared/components/entity-list/entity-list.component';
+@Injectable({
+  providedIn: 'root'
+})
+export class EmailService {
+
+  // TODO: if possible break this file iinto specific HTTP service files for readability
```

Comment:
this is not a large file,, thumb rule is if the number of lines in code is over 200 ish that is when think about breaking dow the file